react-sidebar/react-sidebar-tests.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import * as ReactDom from 'react-dom';
+import Sidebar from 'react-sidebar';
```

Comment:
Similarly, you can use `import Sidebar, { SidebarStyles } from 'react-sidebar'`.